common/core/src/main/java/com/vmware/connectors/common/utils/AuthUtil.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright © 2018 VMware, Inc. All Rights Reserved.
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+package com.vmware.connectors.common.utils;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.security.jwt.JwtHelper;
+
+import java.io.IOException;
+import java.util.Map;
+
+public final class AuthUtil {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+
+    private AuthUtil() {
+        // hide constructor
+    }
+
+    public static String extractUserEmail(String authorization) throws IOException {
```

Comment:
I moved this to a common place because the other Hub connectors will likely need it too.